pkg/services/ssosettings/api/api.go

```diff
@@ -76,17 +76,33 @@ func (api *Api) RegisterAPIEndpoints() {
 	})
 }
 
+// swagger:route GET /v1/sso-settings sso_settings listAllProvidersSettings
```

Comment:
We are missing the swagger parameters from the end of this file for the list endpoint.